src/amoc_test_event.erl

```diff
@@ -0,0 +1,60 @@
+-module(amoc_test_event).
```

Comment:
We need something simpler than that - some sort of check, not event processing. We just need a question "Is the test still running?" to be simply answered with Y/N.